code/__DEFINES/robots.dm

```diff
@@ -66,3 +66,11 @@
 #define BORG_CHOOSE_MODULE_ONE    1
 #define BORG_CHOOSE_MODULE_TWO    2
 #define BORG_CHOOSE_MODULE_THREE   3
+
+#define SKIN_ICON "skin_icon"
+#define SKIN_ICON_STATE "skin_icon_state"
+#define SKIN_PIXEL_X "skin_pixel_x"
+#define SKIN_PIXEL_Y "skin_pixel_y"
+#define SKIN_LIGHT_KEY "skin_light_key"
+#define SKIN_HAT_OFFSET "skin_hat_offset"
+#define SKIN_TRAITS "skin_traits"
```

Comment:
icon, pixel_x, pixel_y and traits were made for downstreams since we dont modify these on any skins currently, i assume these are fine to keep?

Reply:
actually downstreams also change the model selector icon so idk if i should add that too